python/GafferArnoldUI/ArnoldDisplacementUI.py

```diff
@@ -0,0 +1,134 @@
+##########################################################################
+#
+#  Copyright (c) 2016, Image Engine Design Inc. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#      * Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials provided with
+#        the distribution.
+#
+#      * Neither the name of John Haddon nor the names of
+#        any other contributors to this software may be used to endorse or
+#        promote products derived from this software without specific prior
+#        written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import Gaffer
+import GafferArnold
+
+Gaffer.Metadata.registerNode(
+
+	GafferArnold.ArnoldDisplacement,
+
+	"description",
+	"""
+	Creates displacements to be applied to meshes for
+	rendering in Arnold. A displacement consists of a
+	shader to provide the displacement map and several
+	attributes to control the height and other displacement
+	properties.
+
+	Use an ArnoldAttributes node to control the subdivision
+	settings of the mesh, which in turn controls the detail
+	of the displacement. Use a ShaderAssignment node to assign
+	the ArnoldDisplacement to specific objects.
+	""",
+
+	plugs = {
+
+		"map" : [
+
+			"description",
+			"""
+			The Arnold shader that provides the displacement
+			map. Connect a float or colour input to displace
+			along the object normals or a vector input to displace
+			in a specific direction.
+			""",
+
+			"nodule:type", "GafferUI::StandardNodule",
+			"nodeGadget:nodulePosition", "left",
+
+		],
+
+		"height" : [
+
+			"description",
+			"""
+			Controls the amount of displacement. Only used when
+			performing displacement along the normal.
```

Comment:
Hi Lucien,
Yep, I'm sure that would be useful, but this node is just intended to expose the built in displacement parameters on Arnold's polymesh node, and as far as I'm aware there is no such scale parameter. That doesn't stop the user doing a scale inside their shader network though...
Cheers…
John